packages/tldraw/src/lib/ui/components/primitives/TldrawUiSlider.tsx

```diff
@@ -49,9 +52,24 @@ export const TldrawUiSlider = React.forwardRef<HTMLDivElement, TLUiSliderProps>(
 	)
 
 	const handlePointerDown = useCallback(() => {
+		tooltipManager.hideAllTooltips()
 		onHistoryMark('click slider')
 	}, [onHistoryMark])
 
+	// N.B. This is a bit silly. The Radix slider auto-focuses which
+	// triggers TldrawUiTooltip handleFocus when we dbl-click to edit an image,
+	// which in turn makes the tooltip display prematurely.
+	// This makes it wait until we've focused to show the tooltip.
+	useEffect(() => {
+		tltime.setTimeout(
```

Comment:
because this is in react we should use a proper teardown here instead of relying on our custom timer fallback teardown stuff